katas/langtonant/lorena/langtonant.js

```diff
@@ -15,52 +64,31 @@ function mov(x, y, ori) {
     } else {
         ori = (ori + 2) % 4;           //If the ant is on the limit, turn 180° ???
     }
-    movant = new Array (x, y, ori);
-
-    return movant;
+    newPositAnt = new Array (x, y, ori);
+    return newPositAnt;
 }
 
-// Ant´s environment
+// Initial Conditions
+let len = 6;        // Grid´s lenght
+let wid = 6;        // Grid's Width
+let steps = 3;      // Times for tracking
 
-var grid = new Array();
-var len = 6;
-var wid = 6;
+//Start
+var grid = gridInit (len, wid);
+var positAnt = positInit (len, wid);            // first box position
+var x = positAnt[0];
+var y = positAnt[1];
+var ori = positAnt[2];
 
-for (let i = 0; i < len; i++) {
-    grid[i] = new Array();
-    for (let j = 0; j < wid; j++) {
-        grid[i][j] = 0;
-    }
-    console.log(grid[i]);
+while (steps > 0) {
+    ori = newOri(x, y, ori, colorAct);
+    var colorAct = grid[y][x];                  // Actual box color, white 0, black=1
+    var color = colorBox(x, y, colorAct);
+    grid[y][x] = color;
+    positAnt = mov(x, y, ori);
+    x = positAnt[0];
+    y = positAnt[1];
+    steps--;
 }
 
-// Initial Ant´s position
-
-var x = Math.floor(Math.random() * ((wid) - 0) + 0);     // x position random
-var y = Math.floor(Math.random() * ((len) - 0) + 0);      // y position random
-var ori = Math.floor(Math.random() * ((4) - 0) + 0);      // Ant´s orientation Up=0, Right=1, Down=2, Left=3
-
-var movant = new Array(x, y, ori); // firest box position
-
-console.log(movant);
-
-
-var n = 10;                        // Tracking number
-
-while (n > 0) {
-    if (grid[y][x] == 0) {           // Ant´s Orientation
-        ori = (ori + 1) % 4;
-        grid[y][x] = 1;
-    } else {
-        ori = (ori + 3) % 4;
-        grid[y][x] = 0;
-    }
-
-    movant = mov(x, y, ori);
-    x = movant[0];
-    y = movant[1];
-
-    console.log (grid);
-    console.log(movant);
-    n--;
-}
+console.log(grid);
```

Comment:
Once you start adding unit tests, please make sure to remove `console.log` lines so they do not pollute unit test execution output